### PR TITLE
Daccess-417: suppress cite

### PR DESCRIPTION
**Pull request by JeremyDuncan**
## Description of Change:
Removed `Cite` button links and updated tests

### Before
image

### After
image

### CODE REVIEW

**Review by jgreidy — APPROVED** [review]
Surprisingly complex button! Glad to see that code removed.

File: blacklight-cornell/app/views/catalog/_show_tools.html.erb [hunk 1]
```diff
@@ -18,39 +18,6 @@
...
-  <li>
-  <img id="loadingSpinner" style="display: none;" src="/img/ajax-loader.gif">
-  </li>
-  <li class="dropdown">
-    <div class="btn-group">
-      <a href="#" class="btn btn-default btn-sm dropdown-toggle" data-toggle="dropdown" id="cite-menu" aria-haspopup="true" aria-expanded="false"><i class="fa fa-share" aria-hidden="true"></i>Cite <b class="caret"></b></a>
-      <ul class="dropdown-menu" aria-labelledby="cite-menu">
-      <% names = %w[apa chicago cse mla mla8]  %>
-      <%  names.each do |name|  %>
-        <% citation_format = t('blacklight.citation.'+name) %>
```
Comment (chrisrlc) [inline blacklight-cornell/app/views/catalog/_show_tools.html.erb:40]: There's a handful of blacklight.citation.* i18n labels in blacklight.en.yml - can any of those be removed now?

**Comment by chrisrlc** [comment]
I'm not familiar with the book bag, bookmarks, or citation features at all, so I have no idea if this is relevant, but is any of this related to the `bookmarks#show_citation_path` or `book_bags#show_citation_path` route?

File: blacklight-cornell/app/views/catalog/_show_tools.html.erb [hunk 2]
```diff
@@ -18,39 +18,6 @@
 <% if !bookmarks_index %>
 <li><%= render :partial => "bookmark_control" , :locals => {:document => @document} %></li>
 <% end %>
-<!-- TODO: TRANSLATE -->
-<% if citable %>
```
[the remark this answers is not in the document]
Comment (JeremyDuncan) [reply blacklight-cornell/app/views/catalog/_show_tools.html.erb:22]: Nice catch!. should be resolved now

File: blacklight-cornell/app/views/catalog/_show_tools.html.erb (hunk 1 above)
Comment (JeremyDuncan) [reply blacklight-cornell/app/views/catalog/_show_tools.html.erb:40]: these are now removed

File: blacklight-cornell/app/views/catalog/_show_tools.html.erb [hunk 3]
```diff
@@ -18,39 +18,6 @@
...
-  <% citelink_content =  '<i class="fa fa-pencil" aria-hidden="true"></i>' + t('blacklight.tools.cite') %>
-  <li>
-  <%= link_to citation_solr_document_path(:id => @document), {:id => 'citation', :name => 'citationSolr', :class => 'lightboxLink btn btn-default btn-sm', data: {}.merge({ blacklight_modal: "trigger" }) } do %>
-    <%= citelink_content.html_safe %>
-  <% end %>
-  </li>
-<% elsif bookmarks_index || bookbags_index%>
-<%# ************************ %>
-  <li>
-  <img id="loadingSpinner" style="display: none;" src="/img/ajax-loader.gif">
```
[the remark this answers is not in the document]
Comment (JeremyDuncan) [reply blacklight-cornell/app/views/catalog/_show_tools.html.erb:32]: I removed that entire script now

**Pull request merged**